common/dispatchers.hpp

```diff
@@ -48,3 +48,19 @@ extern "C" { \
         } \
    } \
 } \
+
+#define DISPATCH_WITH_BULK_TRANSFER(TYPE, TRANSFER, BULKTRANSFER, MEMBERS) \
```

Comment:
As I understand do you use branch with implementation DISPATCH_WITH_BULK_TRANSFER. I will suggest rebase your branch with last develop: we can get some problems with merge `cyberway.contracts` submodule.